plasma_framework/contracts/src/exits/payment/controllers/PaymentStartInFlightExit.sol

```diff
@@ -213,10 +213,6 @@ library PaymentStartInFlightExit {
             exitData.inputTxs.length == exitData.inFlightTx.inputs.length,
             "Number of input transactions does not match number of in-flight transaction inputs"
         );
-        require(
```

Comment:
ah...okay just realize the same check is done in `getOutputId` so it is hard to do it here.

Might consider to just break this `verifyNumberOfInputsMatchesNumberOfInFlightTransactionInputs` and embed it into `createStartExitData` so that each line immediately verifies the length as that function is sort of not really possible of doing its original purpose.